terraso_backend/apps/project_management/models/projects.py

```diff
@@ -0,0 +1,68 @@
+# Copyright © 2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from apps.core import permission_rules
+from apps.core.models import Group, User
+from apps.core.models.commons import BaseModel
+
+
+class Project(BaseModel):
+    class Meta(BaseModel.Meta):
+        abstract = False
+
+        rules_permissions = {"change": permission_rules.allowed_to_change_project}
```

Comment:
Question: Do we need `delete` here?

Reply:
Yes, but I will leave that for when we implement the feature to delete the project.